graphReconstruct/graphReconstruct.go

```diff
@@ -115,7 +115,17 @@ func SetState(node *expandedTree.ETree, position int) {
 		SetState(node.Left, position)
 		SetState(node.Right, position)
 		if allZero(node.Left.Stored) && allZero(node.Right.Stored) {
-			log.Fatal("no Stored values passed to internal node")
+			// switch statement below accounts for potential deletions
+			// if there is a deletion at current position, set Stored list to equal probabilities
+			isLeftDel := node.Left.Fasta.Seq[position]
+			isRightDel := node.Right.Fasta.Seq[position]
+			switch isLeftDel == 10 && isRightDel == 10 {
+			case true:
+				node.Left.Stored = []float64{0.25, 0.25, 0.25, 0.25}
```

Comment:
Right now setting the base probabilities to 0.25 when there is a gap (missing data) is happening in internal nodes, but I was thinking that would happen at the leaf nodes and then the internal nodes (i.e. recursive step) would not change.

Reply:
Does having a species with missing data (all bases set to 0.25) influence the predictions at ancestral nodes?  For example, if we predict the human chimp ancestor with human, chimp, and gorilla would the predicted human-chimp ancestor be different if I added bonobo with 0.25 for all bases?